### PR TITLE
Update OOPSLA24

### PR DESCRIPTION
**Pull request by only-changer**
### Which conference does this PR update?

- OOPSLA 2024

### Related URL

- https://2024.splashcon.org/track/splash-2024-oopsla#Call-for-Papers
- https://oopsla24.hotcrp.com/deadlines

### CODE REVIEW

File: conference/SE/oopsla.yml [hunk 1]
```diff
@@ -32,5 +32,16 @@
         - deadline: '2023-08-18 23:59:59'
           comment: 'Submission of Revisions Round 2'
       timezone: AoE
-      date: TBD
+      date: October 22-27, 202
```
Comment (ViGeng) [inline conference/SE/oopsla.yml:35]: October 22-27, 2023?

**Review by ViGeng — CHANGES_REQUESTED** [review]
except for the year 2023 typo, the others are fine I think.

File: conference/SE/oopsla.yml (hunk 1 above)
Comment (only-changer) [reply conference/SE/oopsla.yml]: oops, I have fixed that typo.

**Comment by only-changer** [comment]
I have fixed the typo and rebase the repo, please check again

**Review by ViGeng — APPROVED** [review]

File: conference/SE/oopsla.yml [hunk 2]
```diff
@@ -32,5 +32,16 @@
...
       place: Lisbon, Portugal
+    - year: 2024
+      id: oopsla24
+      link: https://2024.splashcon.org/track/splash-2024-oopsla
+      timeline:
+        - deadline: '2023-10-21 7:59:59'
+          comment: 'Submission Deadline Round 1'
+        - deadline: '2024-04-06 7:59:59'
+          comment: 'Submission Deadline Round 2'
+      timezone: EDT
```
Comment (ViGeng) [inline conference/SE/oopsla.yml:45]: please convert it to UTC or AoE.

**Review by ViGeng — CHANGES_REQUESTED** [review]

File: conference/SE/oopsla.yml (hunk 2 above)
Comment (only-changer) [reply conference/SE/oopsla.yml]: fixed, sorry for the inconvenience.

**Review by ViGeng — APPROVED** [review]

**Pull request merged**